packages/react-pdf/src/renderer.js

```diff
@@ -138,8 +146,8 @@ const PDFRenderer = ReactFiberReconciler({
   },
 });
 
-function createPDFInstance(inst) {
-  const { doc, firstPageSkipped } = inst.rootContainerInstance;
+function createPDFInstance(inst, doc) {
+  const { firstPageSkipped } = inst.rootContainerInstance;
   const { children, ...props } = inst.props;
 
   switch (inst.type) {
```

Comment:
Not any more, will spend some time to fix the node renderer, and remove this part of the code.